fractions/javaee/full/no.module.conf

```diff
@@ -0,0 +1 @@
+# This doesn't need module.conf, because it is not a real fraction. It just aggregates fractions on Maven level.
```

Comment:
May be change `This` to `This component` or `This module` ? if you agree then it would apply to to all the similar comments. Np if you prefer to keep it in the current format

Reply:
Yea, good point.

Reply:
Changed to "This fraction".